tests/cross/src/main/scala/tests/BaseCodeActionSuite.scala

```diff
@@ -1,18 +1,18 @@
 package tests
 
-import scala.meta.internal.metals.EmptyCancelToken
 import scala.meta.pc.CancelToken
 import java.nio.charset.StandardCharsets
 import java.nio.file.Files
 import scala.util.control.NonFatal
+import scala.meta.internal.pc.EmptyCancelToken
 
 abstract class BaseCodeActionSuite extends BasePCSuite {
 
   def cancelToken: CancelToken = EmptyCancelToken
 
   def params(
       code: String
-  ): (String, String, Int) = {
+  )(implicit testEnvironment: TestEnvironment): (String, String, Int) = {
```

Comment:
Added `TestEnvironment`, which basically is the context of the test with the PresenationCompiler itself, symbolIndex and the scalaVersion itself.